charts/all-in-one/templates/rudolf-service.yaml

```diff
@@ -63,6 +63,7 @@ spec:
 
 ---
 
+{{- if .Values.rudolfService.db.local }}
```

Comment:
I'm not sure about Helm's syntax, it seems to be inserted when the `...db.local` value is true. But maybe it should be ignored when the value is `true`. 🤔

Reply:
i thought it doesnt need security group when the db is in the same cluster as a pod.

Reply:
Well, then it means the inner values will not be rendered when the `db.local` value is true, right? If then, the value lines should use `{{` instead of `{{-` right?

```
{{- if .Values.rudolfService.db.local }}
apiVersion: v1
kind: Service
metadata:
  name: rudolf-db
  namespace: {{ $.Release.Name }}
  labels:
    app.kubernetes.io/instance: {{ $.Release.Name }}
spec:
  ports:
    - name: "5432"
      port: 5432
      targetPort: 5432
  selector:
    app: rudolf-db
  type: ClusterIP
{{- end }}
```

Reply:
the only difference is that `{{-` doesnt include newline. but as i render, that doesnt make any differ.
let me update it for coherence XD

Reply:
> i thought it doesnt need security group when the db is in the same cluster as a pod.

Wait, then shouldn't the `{{- if .Values.rudolfService.db.local }}` part be `{{- if not .Values.rudolfService.db.local }}` to remove the security group when using local db?

Reply:
aha, i was confused. thank you alot 🙇